models/public/retinaface-resnet50-pytorch/README.md

```diff
@@ -0,0 +1,127 @@
+# retinaface-resnet50-pytorch
+
+## Use Case and High-Level Description
+
+The `retinaface-resnet50-pytorch` model is a PyTorch\* implementation of medium size model with ResNet50 backbone for Face Localization. It can output face bounding boxes and five facial landmarks in a single forward pass. More details provided in the [paper](https://arxiv.org/abs/1905.00641) and [repository](https://github.com/biubug6/Pytorch_Retinaface)
```

Comment:
```suggestion
The `retinaface-resnet50-pytorch` model is a PyTorch\* implementation of medium size RetinaFace model with ResNet50 backbone for Face Localization. It can output face bounding boxes and five facial landmarks in a single forward pass. More details provided in the [paper](https://arxiv.org/abs/1905.00641) and [repository](https://github.com/biubug6/Pytorch_Retinaface)
```

Reply:
Done